src/components/bubble/_bubble.scss

```diff
@@ -14,12 +15,17 @@ $include-html: false !default;
 
   .mint-bubble {
     @include component;
+    display: inline-flex;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    min-height: rhythm(3);
```

Comment:
You need at least 1 for padding and at least 1 for content.
`rhythm(2)` will be great, but it ruins the rhythm;
![](https://i.gyazo.com/9e19343ffcad4b508f8d4018faefdf95.png)